bread/views/browse.py

```diff
@@ -220,6 +226,18 @@ def get_queryset(self):
                 )
         return qs
 
+    @staticmethod
+    def gen_rowclickaction(modelaction):
```

Comment:
I would find it nice to use an enum here.. or at least a docstring that lists the possible modelactions

Reply:
Well, we cannot have a definite list of modelactions because they depend on the model-views that have been added and their names. But right, a doc string would make sense to explain the behaviour.

Reply:
Okay, added a docstring.